src/pages/ArticlePage.jsx

```diff
@@ -19,13 +19,13 @@ function ArticlePage() {
   return isLoading ? (
     <Loading />
   ) : (
-    <article className={styles.ArticlePage}>
-      <h1>{data.article.title}</h1>
-      <h2>Written by {data.article.author}</h2>
+    <article className={styles.Article}>
```

Comment:
Good work so far - see this article for more interesting ways of displaying information: https://www.refactoringui.com/previews/labels-are-a-last-resort